extensions-core/kubernetes-extensions/src/main/java/org/apache/druid/k8s/discovery/K8sDiscoveryModule.java

```diff
@@ -26,6 +26,11 @@
 import com.google.inject.Provider;
 import io.kubernetes.client.openapi.ApiClient;
 import io.kubernetes.client.util.Config;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.List;
+import java.util.TimeZone;
```

Comment:
Please don't change the import order. This is unnecessary. 
And this is not the Druid' code style. 
If you're using intellij, see: https://github.com/apache/druid/blob/master/dev/intellij-setup.md

Reply:
And if Eclipse: https://github.com/apache/druid/blob/master/dev/eclipse_formatting.xml